### PR TITLE
Vend Subscription pipeline sample for Azure DevOps 

### PR DESCRIPTION
**Pull request by Gordonby**
I've added a sample Pipeline that augments an Arm template with passed variable values, which is then added to a new branch and a PR initiated.

Also included a few typo's in the canary multi-environment readme.

### CODE REVIEW

**Comment by Gordonby** [comment]
Any thoughts @ljtill  ?

File: .pipelines/samples/README.md [hunk 1]
```diff
@@ -28,3 +38,38 @@ It's advised to use different Service Principals to access each environment. Eac
 #### Environment names
 
 The terms Canary and Prod are embedded throughout the sample files. Where your Reference Implementation Prefix is different, these replacements will need to be made.
+
+## Subscription-Vend
+
+The Subscription-Vend pipeline demonstrates governed Subscription Creation. It leverages the AzOps deployment capability, through raising a Pull Request on the main branch with a new Arm template dropped at the correct folder scope.
```
Comment (krnese) [inline .pipelines/samples/README.md:44]: Thanks for the work on adding examples for the canary/prod hierarchies. I am a bit hesitant though to elevate subscription creation to be its unique resource, and unique pipeline as it deviate from the generic approach (everything is a resource, and deployment must be invoked at a scope). Could we hold this for and reconsider based on demand and learnings?

Comment (Gordonby) [reply .pipelines/samples/README.md:44]: Totally 👍 
Happy to leave this one here for the time being. Simplifying the subscription creation process (albeit leveraging AzOps) isn't a pattern to be encouraged for other Azure resources.

Aware that there's already several other independent examples of coded/pipeline examples of subscription creation, having one that leverages the AzOps push is going to be preferable where separation is required... :)

**Comment by daltondhcp** [comment]
Closing stale PR for now.

**Pull request closed without merging**